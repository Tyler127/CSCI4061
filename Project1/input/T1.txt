oYuF9jTloU8FNBttHyxN1Fuv50QpqUsoDCfadfRqAaO0TFjfbLADfxzc1zNVg0OE2W4f8hH7uRC1PfK5h92mY5JLe6bYagXB3fn77V2OVlLZ5FaS6GtAekNHearjdNgxPp4giLw125UpgD1qg2A4ilkD9kxNKn6zOoHE5TLqqVzvtoKLtst6oYkfAsrc4Ecqiqc3Zr0gQ7gyEwGkbT6A3xCY61ubRjA1KM4h7t0CkvVNVB70Ly80e1Wui0hYEhOw