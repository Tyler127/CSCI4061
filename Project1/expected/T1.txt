Node 0: 815cd4360810eb318878ad3b9a97b178b7dec37b0f70d23445585813ddf9fd9c
    Node 1: 2d42dd111843ff29d64e2107dc079a39f574536b24245d20b46d28c5fa0f7273
        Node 3: 0536fe7b39b50dd3480e2abf7b4a486dfe992bc120c3a47667df5c9bea3c4cbf
            Node 7: a370ca7bc416e24717207a1dbcb6e668cac904a63963a675995357c6716c5a64
            Node 8: 305d8c2db87b9e30c8a53756802d7e019c06096e046863abe0c743e2ba36f1ea
        Node 4: 139743f1e53c7d1e4df7f87955e339edd06e97d0572d69abe90f3bc05652976b
            Node 9: be27603b73c96b6caef33026dff1879a5eb4eb9368298bd9913baefeb68e57b4
            Node 10: 0818a439ca8cf5935e0a5b75508661959fbecbf150f18354a15bb3b939393db7
    Node 2: ee90c624ef47bcda918d1403f1a63afb437e94d507f9d2997d86816ff5c24bca
        Node 5: 89d702cd6a6f4d8c9046936e8aeef9b9253ca9cffd287fd41cca5af80b8f8b67
            Node 11: 96c9928bac6449c36824a9320005640176ce59c98de5667f3673ade6732344df
            Node 12: 5230513745cbd0e31b27bf97659a265e529e57b98bc14a682da136fc23d4d19f
        Node 6: 744aac89e0b63f2fb4406ed4185ba3ce9631430a6a37c5eee003f9adc6913530
            Node 13: ce0a8fa74bf23e2362ec6caed748fb8da027622de48928af885fa41228f1de61
            Node 14: d11c3767006a50d51fec4a7bfb52b66ddbd2fd953e15347c2352050d8fb92cba
